lib/local-links-manager/import/enabled_service_checker.rb

```diff
@@ -0,0 +1,38 @@
+module LocalLinksManager
+  module Import
+    class EnabledServiceChecker
+      CSV_URL = "https://raw.githubusercontent.com/alphagov/publisher/master/data/local_services.csv"
+
+      def self.enable
+        new.enable_services
+      end
+
+      def initialize(csv_downloader = CsvDownloader.new(CSV_URL))
+        @csv_downloader = csv_downloader
+      end
+
+      def enable_services
+        @csv_downloader.each_row do |row|
+          check_enabled_services(row)
+        end
+      rescue CsvDownloader::Error => e
+        Rails.logger.error e.message
+      rescue => e
+        Rails.logger.error "Error #{e.class} enabling in #{self.class}\n#{e.backtrace.join("\n")}"
+      end
+
+    private
+
+      def check_enabled_services(row)
```

Comment:
Quite possibly nitpicking, but I'm wondering whether this method could be renamed to `enable_service`? Singular as opposed to plural because we're only dealing with one service at a time. And removing the 'check' part of the name as it does not express the intent entirely.